M2/Macaulay2/packages/SpectralSequences.m2

```diff
@@ -249,21 +148,20 @@ min FilteredComplex := K -> min spots K
 support FilteredComplex := List => (
      K -> sort select (spots K, i -> K#i != 0))
 
-
 FilteredComplex _ InfiniteNumber :=
-FilteredComplex _ ZZ := ChainComplex => (K,p) -> (
+FilteredComplex _ ZZ := Complex => (K,p) -> (
   if K#?p then K#p 
   else if p < min K then K#(min K) 
   else if p > max K then K#(max K)
   )
 
 FilteredComplex ^ InfiniteNumber :=
-FilteredComplex ^ ZZ := ChainComplex => (K,p) -> K_(-p)
+FilteredComplex ^ ZZ := Complex => (K,p) -> K_(-p)
 
 chainComplex FilteredComplex := ChainComplex => K -> K_infinity
```

Comment:
There are still a number of `chainComplex` / `ChainComplexes` references (like this line) that should be updated to `complex` / `Complex`.